lib/engine/game/g_1848/step/track.rb

```diff
@@ -9,6 +9,12 @@ module G1848
       module Step
         class Track < Engine::Step::Track
           include Engine::Game::G1848::Tracker
+
+          def actions(entity)
+            return super unless entity == @game.boe
```

Comment:
You can also create a module for boe to skip all the steps it shouldn't run rather than repeating the code.

Reply:
do you know of a working example of this?

Reply:
module skip_boe
  def actions(entity)
     return [] if @game.boe == entity

     super
  end
end

Then in the step:
include skip_boe